src/walletConnect/saga.ts

```diff
@@ -531,6 +531,13 @@ function* handleAcceptRequest({ request }: AcceptRequest) {
       ...defaultTrackedProperties,
       error: e.message,
     })
+
+    // Notify the client about the error if possible
+    const { topic, id } = request
+    if (client && topic && id) {
+      const response = formatJsonRpcResult(id, { error: e })
+      yield* call([client, 'respondSessionRequest'], { topic, response })
+    }
```

Comment:
Great!  
Could we wrap this in a `try / catch` otherwise if an exception is thrown `handlePendingStateOrNavigateBack` will never be called.